system/Debug/Toolbar/Collectors/Database.php

```diff
@@ -76,6 +76,8 @@ public function __construct()
      * data.
      *
      * @internal param $ array \CodeIgniter\Database\Query
+     *
+     * @return void
```

Comment:
The docblock above is cryptic to me but if this method is `@internal` then it is safe to apply the return natively. @kenjis  Do you know what that tag means?

Reply:
I don't know. Typo?
It came from: https://github.com/codeigniter4/CodeIgniter4/commit/3efa4667#diff-48dee7e39770b002fe3f48f70db03aaa01fd79d5830d5ee91ee9f8ccda44b4eaR109

Reply:
I sent PR to remove it. #7798